tests/components/light/test_intent.py

```diff
@@ -1,29 +1,37 @@
 """Tests for the light intents."""
 from homeassistant.components import light
 from homeassistant.components.light import intent
-from homeassistant.const import ATTR_ENTITY_ID, ATTR_SUPPORTED_FEATURES, SERVICE_TURN_ON
+from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.const import ATTR_SUPPORTED_FEATURES
+from homeassistant.const import SERVICE_TURN_ON
 from homeassistant.helpers.intent import IntentHandleError
-
 from tests.common import async_mock_service
 
 
 async def test_intent_set_color(hass):
     """Test the set color intent."""
-    hass.states.async_set(
-        "light.hello_2", "off", {ATTR_SUPPORTED_FEATURES: light.SUPPORT_COLOR}
-    )
+    hass.states.async_set("light.hello_2", "off",
+                          {ATTR_SUPPORTED_FEATURES: light.SUPPORT_COLOR})
     hass.states.async_set("switch.hello", "off")
     calls = async_mock_service(hass, light.DOMAIN, light.SERVICE_TURN_ON)
     await intent.async_setup_intents(hass)
 
     result = await hass.helpers.intent.async_handle(
         "test",
         intent.INTENT_SET,
-        {"name": {"value": "Hello"}, "color": {"value": "blue"}},
+        {
+            "name": {
+                "value": "Hello"
+            },
+            "color": {
+                "value": "blue"
+            }
+        },
     )
     await hass.async_block_till_done()
 
-    assert result.speech["plain"]["speech"] == "Changed hello 2 to the color blue"
+    assert result.speech["plain"][
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)